### ISSUE TITLE
Nothing to choose on start page

### ISSUE THREAD

**Issue by geocachermgo** [cgeo/cgeo#12010]
### Describe your problem!

I opened cgeo or went back to the start page and there were no menu entries at the bottom.

### How to reproduce?

I had it more than once but found now way to reproduce it.

### Expected result after these steps?

Menu should always avalaible. Not like here:
![Screenshot_2021-10-28-14-42-25-498_cgeo geocaching](https://user-images.githubusercontent.com/43813636/139258728-fb610d0a-6534-4fab-baa5-cc70880fe5f8.jpg)

### Reproducible

Unclear

### c:geo Version

2021.10.28-NB-1ac0319

### System information

```text
System information

c:geo version: 2021.10.28-NB-1ac0319

Device:

Device type: MI 8 (dipper, Xiaomi)
Available processors: 8
Android version: 10
Android build: QKQ1.190828.002 test-keys
Screen resolution: 1080x2029px (392x737dp)
Pixel density: 2.75
System font scale: 1.0 / used scale: 1.0
Sailfish OS detected: false
Google Play services: disabled - 21.39.17 (120400-405802548)

Sensor and location:

Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector

Program settings:

Settings: v8, Count:167
Set language: de_DE (system default)
System date format: dd.MM.yy
Time zone: GMT+02:00
Debug mode active: no
Last backup: 28. Okt., 14:29
Routing mode: Bike
Live map mode: true
OSM multi-threading: true / threads: 4
Map: Kombiniert (Offline)
Id: cgeo.geocaching.maps.mapsforge.MapsforgeMapProvider$OfflineMultiMapSource
Atts: austria.map: / mapsforge-map-writer-master-SNAPSHOT /  / baden-wuerttemberg.map: / Map data (c) OpenStreetMap contributors /  / bulgaria.map: / mapsforge-map-writer-master-SNAPSHOT /  / france.map: / mapsforge-map-writer-0.6.0-rc3 /  / germany.map: / Map data (c) OpenStreetMap contributors /  / great_britain.map: / mapsforge-map-writer-0.3.1-SNAPSHOT /  / greece_v5.map: / mapsforge-map-writer-0.10.0 /  / india.map: / mapsforge-map-writer-0.5.1-rc4 /  / ireland.map: / mapsforge-map-writer-0.3.1-SNAPSHOT /  / norway.map: / mapsforge-map-writer-0.10.0 /  / romania.map: / mapsforge-map-writer-master-SNAPSHOT /  / slovenia.map: / mapsforge-map-writer-0.6.0-rc3 /  / switzerland.map: / Map data (c) OpenStreetMap contributors /  / world.map: / mapsforge-map-writer-0.5.0-rc4
Theme: none

Filters:

Hide waypoints: visited
LIVE: Status:Gefunden=Nein ([:inconclusive=false:advanced=true]AND(status:found_no))
OFFLINE: Keine ([:inconclusive=false:advanced=true])

Services:

Geocaching sites enabled: geocaching.com: Logged in (Anmeldung OK) / PREMIUM
Geocaching.com date format: dd.MM.yy
Routing: external / BRouter installed: true
Installed c:geo plugins:  none

Permissions & paths:

Fine location permission: granted
Write external storage permission: granted
System internal c:geo dir: /data/user/0/cgeo.geocaching (80,6 GB free) v2 internal isDir(6 entries)
Legacy User storage c:geo dir: /storage/emulated/0/cgeo (80,6 GB free) v2 external non-removable isDir(8 entries)
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (80,6 GB free) v2 external non-removable isDir(743 entries)
Internal theme sync (is turned off): /data/user/0/cgeo.geocaching/MapThemeData (80,6 GB free) v2 internal isDir(0 entries)
Public Folders: #11
BASE: /cgeo (User-Defined)[/cgeo[DOCUMENT#0:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo, Av:true, files:>=7, dirs:>=12, totalFileSize:>=81,2 KB, free space: 80,6 GB, files on device: 7413760)
OFFLINE_MAPS: [Legacy]/storage/emulated/0/Locus/mapsVector (User-Defined)[/storage/emulated/0/Locus/mapsVector[FILE#0:p-file:///storage/emulated/0/Locus/mapsVector::]] (Uri: file:///storage/emulated/0/Locus/mapsVector, Av:true, files:14, dirs:1, totalFileSize:7,1 GB, free space: 80,6 GB, files on device: -1)
OFFLINE_MAP_THEMES: [Legacy]/storage/emulated/0/Locus/mapsVector/_themes (Default)[/storage/emulated/0/Locus/mapsVector/_themes[PERSISTABLE_FOLDER(OFFLINE_MAPS)#1:p-file:///storage/emulated/0/Locus/mapsVector::/_themes]] (Uri: file:///storage/emulated/0/Locus/mapsVector/_themes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 80,6 GB, files on device: -1)
LOGFILES: /cgeo/logfiles (Default)[/cgeo/logfiles[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/logfiles]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Flogfiles, Av:true, files:5, dirs:0, totalFileSize:522,5 KB, free space: 80,6 GB, files on device: 7413760)
GPX: /cgeo/gpx (Default)[/cgeo/gpx[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/gpx]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fgpx, Av:true, files:56, dirs:0, totalFileSize:16,6 MB, free space: 80,6 GB, files on device: 7413760)
BACKUP: /cgeo/backup (Default)[/cgeo/backup[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/backup]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Fbackup, Av:true, files:>=10, dirs:>=10, totalFileSize:>=133,6 MB, free space: 80,6 GB, files on device: 7413760)
FIELD_NOTES: /cgeo/field-notes (Default)[/cgeo/field-notes[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/field-notes]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Ffield-notes, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 80,6 GB, files on device: 7413760)
SPOILER_IMAGES: /cgeo/GeocachePhotos (Default)[/cgeo/GeocachePhotos[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/GeocachePhotos]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2FGeocachePhotos, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 80,6 GB, files on device: 7413760)
ROUTING_BASE: /cgeo/routing (Default)[/cgeo/routing[PERSISTABLE_FOLDER(BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/routing]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Frouting, Av:true, files:7, dirs:1, totalFileSize:81,2 KB, free space: 80,6 GB, files on device: 7413760)
ROUTING_TILES: /cgeo/routing/segments4 (Default)[/cgeo/routing/segments4[PERSISTABLE_FOLDER(ROUTING_BASE)#1:p-content://com.android.externalstorage.documents/tree/primary%3Acgeo::/routing/segments4]] (Uri: content://com.android.externalstorage.documents/tree/primary%3Acgeo/document/primary%3Acgeo%2Frouting%2Fsegments4, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 80,6 GB, files on device: 7413760)
TEST_FOLDER: [Legacy]/data/user/0/cgeo.geocaching/files/unittest (Default)[/data/user/0/cgeo.geocaching/files/unittest[FILE#1:p-file:///data/user/0/cgeo.geocaching/files::/unittest]] (Uri: file:///data/user/0/cgeo.geocaching/files/unittest, Av:true, files:0, dirs:0, totalFileSize:0 B, free space: 80,6 GB, files on device: -1)
Map render theme path:
PersistedDocumentUris: #1
TRACK: null
Persisted Uri Permissions: #1
content://com.android.externalstorage.documents/tree/primary%3Acgeo (30. Jan., 13:54):RW
Database: /data/user/0/cgeo.geocaching/databases/data (v98, Size:32,2 MB) on system internal storage

--- End of system information ---
```

**Comment by fm-sys** [comment]
Now I experienced this as well for the first time. Pressing the system back button does solve the problem, so it seems to be caused by the fact that the "start" page isn't launched as top-level. Other activities will have the back icon in the action bar in that case, but for the "start" page, we explicitly overwrite that icon with the c:geo logo. 

> When pressing the back button of my device it (most of the time) returns to the last active activity (live map in this example).

> I still suspect some activity mixup or suspend/resume issue as each time I am in such situation pressing the back key brings me back to some other (last opened?) part of the app and pressing back multiple times at the end brings me back to a working dashboard. 

_Originally posted by @Lineflyer in https://github.com/cgeo/cgeo/issues/11908#issuecomment-947613813_

I will provide a fix...

**Comment by fm-sys** [comment]
Finally, I found a way to easily reproduce this issue. 

Tap two times quickly onto start - here we go - nothing to choose on start page anymore...

![grafik](https://user-images.githubusercontent.com/64581222/140725664-d4f814e5-78d1-4549-9e51-620715fdfa4f.png)

**Comment by fm-sys** [comment]
For my tests, it seems to be fixed with `2021.11.08-NB1-d4b1a3e`. Feel free to report back if someone still has trouble...
